src/manifest.chrome.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Copy as Markdown",
-  "version": "2.1.0",
-  "version_name": "2.1.0",
+  "version": "2.1.1",
+  "version_name": "2.1.1",
```

Comment:
I prefer not updating version until I manually jump version and release it. Would be appreciated if you revert version changes.